---
title: Epigenomics
type: blog
menu:
  main:
    parent: data_analysis
    identifier: epigenomics
    weight: 3
    # params:
    #   icon:
    #     vendor: bootstrap
    #     name: boxes
---

Epigenomic modifications are modifications that affect the genome without modifying the underlying DNA sequence. These can be chemical modifications such as DNA methylation (= addition of a methyl group the 5th C of the cytosine base) or proteins (and their modifications) binding to the DNA that affect the DNA structural properties (packaging, 3D structure), for example. Ultimately, epigenomic modifications are thought to affect the DNAâ€™s transcriptional potential (chromatin state), or in other words, regulate gene expression.

Epigenomic modifcations are used as biomarkers and provide insight into biological mechansim and processed.

## DNA methylation

We analyse DNA methylation mainly from the Illumina Infinium beadchip microarray (human, mouse), but also from sequenced based technologies such as whole genome bisulfite sequencing, reduced representation bisulfite sequencing and bisulfite free tequnologies such as enzymatic Methyl-seq.

### Our analysis:

- Differential methylation analysis

- Functional annotation of differential methylated CpGs and differentially methylated regions (associated genes, overlapping regulatory elements, CGI context)

- Identification of different types of methylation regions: unmethylated regions (UMRs), hypermethylated regions (UMRs), partially methylated domains (PMDs)

- Cell type deconvolution

- Age prediction

- CNV prediction

## Chromatin

We process mainly sequencing data from Chromatin Immunopreciptation (CHIP-seq) to study protein binding (histones, transcription factors) and Assay for Transposase-Accessible Chromatin (ATAC-seq) to identify open chromatin regions.

### Our analysis:

- Peak calling

- Functional annotation peaks (genes, regulatory)

- Motif analysis

- Differential analysis